src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -92,35 +95,36 @@ def ingest_tiff_udf(
     task_prefix = f"{dag_name} - Task"
 
     logger.info("Building graph")
-    graph = tiledb.cloud.dag.DAG(
+    graph = dag.DAG(
         name=dag_name,
-        mode=tiledb.cloud.dag.Mode.BATCH,
+        mode=dag.Mode.BATCH,
         max_workers=max_workers,
         namespace=namespace,
     )
+    submit = graph.submit_local if local else graph.submit
```

Comment:
If you’re running a graph server-side, then `submit_local` will never work; you will have to configure that also.

Reply:
What do you mean by server-side? This functionality is introduced the same way we use it here https://github.com/TileDB-Inc/TileDB-Cloud-Py/blob/6ca6404280758982606546df11f633a7494e8c5f/src/tiledb/cloud/vcf/ingestion.py#L920

Reply:
"batch mode", i.e. the coordinator which runs the graph is on the server, rather than on the client. a key difference in that code is:

https://github.com/TileDB-Inc/TileDB-Cloud-Py/blob/6ca6404280758982606546df11f633a7494e8c5f/src/tiledb/cloud/vcf/ingestion.py#L910-L920